<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IlastikShell</string>
  </property>
  <widget class="QWidget" name="mainLayout">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="startupPage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticallayout">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>410</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>410</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="1" column="1">
              <layout class="QGridLayout" name="gridlayout">
               <item row="1" column="0">
                <widget class="QScrollArea" name="OpenList">
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents_2">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>678</width>
                    <height>199</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <item>
                    <layout class="QVBoxLayout" name="VL2">
                     <item>
                      <widget class="QLabel" name="label2">
                       <property name="text">
                        <string>Open Project</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="browseFilesButton">
                       <property name="text">
                        <string>Browse Files</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="Line" name="line">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QScrollArea" name="CreateList">
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>678</width>
                    <height>199</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <layout class="QVBoxLayout" name="VL1">
                     <item>
                      <widget class="QLabel" name="label1">
                       <property name="text">
                        <string>Create New Project</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>288</width>
                 <height>410</height>
                </size>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap>../ilastik-logo-alternate-colors.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="contentPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QSplitter" name="sideSplitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="childrenCollapsible">
              <bool>false</bool>
             </property>
             <widget class="QSplitter" name="mainSplitter">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QTreeWidget" name="appletBar">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="Highlight">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>62</red>
                     <green>62</green>
                     <blue>62</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="Highlight">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>62</red>
                     <green>62</green>
                     <blue>62</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="Highlight">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>240</red>
                     <green>240</green>
                     <blue>240</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
               <property name="lineWidth">
                <number>2</number>
               </property>
               <property name="midLineWidth">
                <number>2</number>
               </property>
               <property name="alternatingRowColors">
                <bool>false</bool>
               </property>
               <property name="indentation">
                <number>15</number>
               </property>
               <property name="rootIsDecorated">
                <bool>true</bool>
               </property>
               <property name="headerHidden">
                <bool>true</bool>
               </property>
               <column>
                <property name="text">
                 <string notr="true">Task</string>
                </property>
               </column>
              </widget>
              <widget class="QWidget" name="layoutWidget">
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="imageSelectionGroup" native="true">
                  <layout class="QHBoxLayout" name="imageSelectionGroup_layout">
                   <item>
                    <widget class="QLabel" name="currentViewLabel">
                     <property name="text">
                      <string>Current View:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="imageSelectionCombo">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QStackedWidget" name="viewerControlStack">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                    <horstretch>2</horstretch>
                    <verstretch>2</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>300</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Plain</enum>
                  </property>
                  <property name="lineWidth">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
             <widget class="QStackedWidget" name="appletStack">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>2</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>500</width>
                <height>100</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>500</width>
                <height>100</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
